Makefile

```diff
@@ -64,20 +64,20 @@ start-idp:  ## 🏃 Start the idp for testing jwt
 # Start hosting the application using `sandbox.sh` and enable custom JWT authentication
 start-host: stop-host  ## 🏃 Start the CCF network using Sandbox.sh
 	@echo -e "\e[34m$@\e[0m" || true
-	MEMBER_COUNT=${MEMBER_COUNT} source ./scripts/ccf/sandbox_local/up.sh --build && \
-	source ./scripts/kms/js_app_set.sh && \
-	source ./scripts/kms/constitution_set.sh \
+	MEMBER_COUNT=${MEMBER_COUNT} . ./scripts/ccf/sandbox_local/up.sh --build && \
```

Comment:
Have you go something against 'source" vs '.'? Or is there a particular difference I am missing?

Reply:
Nothing major, `.` is POSIX compliant whereas `source` is exclusive to bash and I think the code looks nicer personally